runtime/drivers/postgres/postgres.go

```diff
@@ -15,6 +15,37 @@ import (
 
 func init() {
 	drivers.Register("postgres", driver{})
+	drivers.RegisterAsConnector("postgres", driver{})
+}
+
+var spec = drivers.Spec{
+	DisplayName: "Postgres",
+	Description: "Connect to Postgres.",
+	SourceProperties: []drivers.PropertySchema{
+		{
+			Key:         "sql",
+			Type:        drivers.StringPropertyType,
+			Required:    true,
+			DisplayName: "SQL",
+			Description: "Query to extract data from Postgres.",
+			Placeholder: "select * from table;",
+		},
+		{
+			Key:         "pg_database_url",
+			DisplayName: "Postgress Connection String",
+			Type:        drivers.StringPropertyType,
+			Required:    false,
+			Href:        "https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING",
+			Placeholder: "postgresql://user:password@localhost/mydb",
+			Hint:        "Either set this or configure PG_DATABASE_URL env variable.",
+		},
+	},
+	ConfigProperties: []drivers.PropertySchema{
+		{
+			Key:    "pg_database_url",
+			Secret: true,
+		},
+	},
```

Comment:
The `pg_` prefix feels redundant here, since it's in the scope of a `type: postgres`. Would suggest just `database_url`

Reply:
I had prefixed it with `pg` to keep it separate from `database_url` for other sql drivers that we may support in future. But given we plan to expose `connectors.postgres.database_url` to users as you indicated in other comments we can change it to `database_url`.

Reply:
Okay, I think prefixing is better – leaning into named connectors also enable e.g. pulling data from two different Postgres databases into the same project